language/unit_tests/test_client.py

```diff
@@ -17,28 +17,33 @@
 
 class TestClient(unittest.TestCase):
 
-    def _getTargetClass(self):
+    @staticmethod
+    def _get_target_class():
```

Comment:
1. Will do it with a script
1. I am unclear why it is of debatable value. PEP8 exists.